Sprintfinity3902/Collision/CollisionDetector.cs

```diff
@@ -12,6 +12,10 @@ namespace Sprintfinity3902.Collision
 {
     public class CollisionDetector {
 
+        private static int LEFT = 1;
+        private static int RIGHT = 2;
+        private static int UP = 3;
+        private static int DOWN = 4;
```

Comment:
Could do this via an enum or using one of them that's already set up, fine though.